src/components/NcModal/NcModal.vue

```diff
@@ -214,10 +217,10 @@ export default {
 			tabindex="-1">
 			<!-- Header -->
 			<transition name="fade-visibility" appear>
-				<div class="modal-header">
+				<div class="modal-header" data-theme-dark>
```

Comment:
This doesn't seem to work if a theme is enforced :warning:

Reply:
That requires a fix on server I think, to ensure we at least load the dark theme with data attribute if another one is enforced :thinking:

Reply:
https://github.com/nextcloud/server/pull/46504

Reply:
But then talk has a problem as they are using this to have the dark "chat view" on every theme

Reply:
Yes, that's k, this is an edge case :)
Hope the server fix suits you! :+1: